src/lib/addressbook.ts

```diff
@@ -16,3 +20,18 @@ export function getChainWNativeTokenAddress(chain: Chain): Hex {
     const tokens = addressbook.addressBook[chain].tokens;
     return tokens.WNATIVE.address as Hex;
 }
+
+export function getChainFeesTokenDecimals(chain: Chain): number {
+  const tokens = addressbook.addressBook[chain].tokens;
+  return tokens.FEES.decimals;
```

Comment:
need either a fallback to WNATIVE or update of the addressbook version

Reply:
sorry yeah, mentioned on discord but not in prs, these are all drafts as need to publish new version of address book first then update in all the apis